### DISCUSSION TITLE
GDMO IS A TROJAN VIRUS

### DISCUSSION THREAD

**Discussion by DXBOSS12** [maxnut/GDmenu#178] [General]
I'm Not Sure if this is right but after trying to download the latest version of the GDMO my Windows Security stopped the download. I went on VirusTotal to scan it and it is flagged as a **Trojan Virus**. This never happend before just in the latest version (compatible with 2.204).
![Screenshot 2024-01-17 201235](https://github.com/maxnut/GDMegaOverlay/assets/156956886/3e00c985-52b3-4125-8387-533d556f6fcf)

**Comment by acer488** [comment]
GDMO injects itself into Geometry Dash, which is what a trojan does. GDMO isn't a virus, it's a false flag.

**Comment by MysticAx0lotl** [comment]
It's pretty hard to hide malware in open source code my guy
